queasars/minimum_eigensolvers/evqe/evqe.py

```diff
@@ -87,6 +87,14 @@ class EVQEMinimumEigensolverConfiguration:
         a python ThreadPool executor. If a dask Client is used, both the Sampler and Estimator need to be serializable
         by dask, otherwise the computation will fail. If no parallel_executor is provided a ThreadPoolExecutor
         with as many threads as population_size will be launched
+    :param use_tournament_selection: indicates whether to use tournament selection. By default, this is
+        set to False. In that case, roulette wheel selection is used. Should be true, if the measured expectation
+        values can be negative.
```

Comment:
Why do negative expectation values cause trouble for roulette wheel selection (that's how I read the description)? I thought the probabilities are taken after a normalization to [0, 1]?

Reply:
It does not cause trouble per se. The code will still work, but since we shift all values so that none are negative, this might squeue the relation and thus the selection likelihood of the fitnesses. Take the fitnesses -1000, 100, 1000.  The fitness 100 is 10 times better than 1000, right? If we shift all values by 1001, we get 1, 1100, 2000. Now the second fitness is only two times better than the third. This is  an expected consequence of the normalization here. But it might not be, what the user wants. As tournament selection does not encounter such issues due to its ordinal nature, its usage is recommended here. Should i weaken the recommendation in the docstring, so that it is clear, that roulette wheel selection will still work?

Reply:
Ok, understood. Thank you!